notion/client.py

```diff
@@ -26,14 +26,15 @@ class NotionClient(object):
     for internal use -- the main one you'll likely want to use is `get_block`.
     """
 
-    def __init__(self, token_v2, start_monitoring=True, cache_key=None):
+    def __init__(self, token_v2, monitor=True, start_monitoring=True, cache_key=None):
         self.session = Session()
         self.session.cookies = cookiejar_from_dict({"token_v2": token_v2})
         cache_key = cache_key or hashlib.sha256(token_v2.encode()).hexdigest()
         self._store = RecordStore(self, cache_key=cache_key)
-        self._monitor = Monitor(self)
```

Comment:
It's a good practice to set the variables to None if they are not used. So, I would suggest to set `self._monitor` to `None` if `monitor` flag is off (right now, it won't be set at all).

This way, all code can assume that the attribute `_monitor` exists, and work with that. No clumsy checks like `if hasattr(self, '_monitor')` won't be required anywhere in the future.